channeldb/migration_01_to_11/meta.go

```diff
@@ -0,0 +1,78 @@
+package migration_01_to_11
+
+import "github.com/coreos/bbolt"
+
+var (
+	// metaBucket stores all the meta information concerning the state of
+	// the database.
+	metaBucket = []byte("metadata")
+
+	// dbVersionKey is a boltdb key and it's used for storing/retrieving
+	// current database version.
+	dbVersionKey = []byte("dbp")
+)
+
+// Meta structure holds the database meta information.
+type Meta struct {
+	// DbVersionNumber is the current schema version of the database.
+	DbVersionNumber uint32
+}
+
+// FetchMeta fetches the meta data from boltdb and returns filled meta
+// structure.
+func (d *DB) FetchMeta(tx *bbolt.Tx) (*Meta, error) {
```

Comment:
Is this code used by anything now?

Also I wouldn't mind removing tests, since they could depend on code that otherwise could be deleted.

Reply:
The code is used to do `syncVersion` for migration tests. But that isn't really necessary to test every time. Added a commit that removes it.

Non-migration tests are all removed (afaik), the `_test.go` files that remain just contain some vars and helpers that are used in the migration tests.